### PR TITLE
Add supporting documentation for RESTEasy 7.0.0.Alpha1 TCK testing fo…

### PR DESCRIPTION
**Pull request by jamezp**
…r Jakarta REST 4.0.

### CODE REVIEW

File: jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-17.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
+= TCK Results
+:ext-relative: {outfilesuffix}
+:imagesdir: ../../images/
+
+image:resteasy_logo_100x.png[RESTeasy, align="center"]
+
+[[introduction]]
+== Introduction
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], the
```
Comment (scottmarlow) [inline jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-17.adoc:9]: Another reference for requirements is the `How to Get Listed as a Compatible Product` section on https://jakarta.ee/compatibility/get-listed.  I do like your reference to https://www.eclipse.org/legal/tck.php as well.

File: jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-21.adoc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+* Specification Name, Version and download URL:
+https://jakarta.ee/specifications/restful-ws/4.0/[Jakarta RESTful Web Services 4.0]
+
+* TCK Version, digital SHA-256 fingerprint and download URL:
+https://download.eclipse.org/ee4j/restful-ws/tck/eftl/jakarta-restful-ws-tck-4.0.0.zip[Jakarta RESTful Web Services TCK 4.0.0], SHA-256: 9494caaf67a3e5524a5ee1d6e4b749782ca1cbc920c6451583b6249e51c4ac18 ]
+
+* Any Additional Specification Certification Requirements: None
+
+* Public URL of TCK Results Summary: link:tck-results/jdk-21.txt[RESTEasy 7.0.0.Alpha1 TCK results]
```
Comment (scottmarlow) [inline jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-21.adoc:45]: For other easy reference, the tck results are at https://github.com/resteasy/certifications/blob/main/jakarta-rest-4.0/7.0.0.Alpha1/tck-results/jdk-21.txt

File: jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-17.adoc (hunk 1 above)
Comment (scottmarlow) [reply jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-17.adoc:9]: FYI https://jakarta.ee/committees/specification/tckprocess/ also describes certification requirements as well.

File: jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-21.adoc (hunk 2 above)
Comment (jamezp) [reply jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-21.adoc:45]: Yeah, the adoc file should link to that file.

File: jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-17.adoc (hunk 1 above)
Comment (jamezp) [reply jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-17.adoc:9]: Okay, should I update it?

Comment (scottmarlow) [reply jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-17.adoc:9]: No, sorry to not be clear enough.  Just wanted to point out the process descriptions on the other two links.

Comment (jamezp) [reply jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-17.adoc:9]: No worries, it's good to know!

**Pull request merged**